### DISCUSSION TITLE
Does tick means it is enabled or disabled on "Privacy"?

### DISCUSSION THREAD

**Discussion by LorenzoPonteggia** [Sophia-Community/SophiApp#91] [Q&A] [answered]
Hello,

Dumb question, but, does ticking the box on the "diagnostic tracking tasks" in the privacy section means that they are disabled or enabled?

I would assume that having all of them unticked means that they are disabled, or ticking them and then running the app means that they have been disabled?

Many Thanks,

![Screenshot (1)](https://user-images.githubusercontent.com/83926649/178104699-f9b7473c-3c32-4715-8b22-7cac76b35a60.png)

**Comment by farag2** [comment, marked as answer]
If check is set means that this feature is enabled in OS. So uncheck them all and apply.

**Reply by farag2** [reply]
So everything is like in any ather app. Wording means something in positive way. Unchecking it will disable.
